src/Libraries/RevitNodes/Elements/Element.cs

```diff
@@ -761,6 +761,23 @@ private IEnumerable<Element> GetElementSubComponents(Autodesk.Revit.DB.Element e
             return components;
         }
 
+        /// <summary>
+        /// Returns the type Element with the given name.
+        /// </summary>
+        /// <param name="name">Name of the type</param>
+        /// <returns>Type Element</returns>
+        public static Element ElementTypeByName(string name)
+        {
+            if (String.IsNullOrWhiteSpace(name))
+                throw new ArgumentNullException(nameof(name));
+
+            var elementType = DocumentManager.Instance
+                .ElementsOfType<Autodesk.Revit.DB.ElementType>()
+                .FirstOrDefault(x => x.Name == name).ToDSType(true);
```

Comment:
`FirstOrDefault` returns `null` if it doesn't find anything, so the next chained method `ToDSType` could fail.

If we don't find anything, we should `throw NotFoundException`, which then makes it safe to move `.ToDSType` to the return statement

Reply:
Also, `ToDSType` should normally be on the next line since it's the next method in the chain